tests/dummy/app/router.js

```diff
@@ -6,6 +6,7 @@ const Router = EmberRouter.extend({
   rootURL: config.rootURL
 });
 
+// eslint-disable-next-line @typescript-eslint/no-empty-function
 Router.map(function() {
 });
```

Comment:
FWIW, I don't know that we actually need to have this. Probably can just delete it?